xml/src/main/java/org/ehcache/xml/Conversions.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ehcache.xml;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Provides XML model to Java object conversions.
+ */
+public final class Conversions {
+
+  /**
+   * Private, niladic constructor to prevent instantation.
+   */
+  private Conversions() {
+  }
+
+  /**
+   * Converts {@link org.ehcache.xml.model.TimeUnit} to Java {@link TimeUnit}.
+   *
+   * @param unit the XML model time unit to convert
+   *
+   * @return the Java {@code TimeUnit} corresponding to {@code unit}
+   */
+  public static TimeUnit toJavaTimeUnit(org.ehcache.xml.model.TimeUnit unit) {
```

Comment:
If we're moving this to allow you to reuse core parsing logic, maybe other core parsing logic should be similarly exposed.  Also maybe we should use these methods to encapsulate the JAXB usage as well?